source/guide_bitwarden.rst

```diff
@@ -0,0 +1,303 @@
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://github.com/dani-garcia/bitwarden_rs.git
+ 
+In order to build bitwarden_rs  sucessfully you'll need to set an environment variable pointing to the sqlite3 header files:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ export SQLITE3_LIB_DIR=/var/lib64
+
+``cd`` into the automatically created folder:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd bitwarden_rs
+ [isabell@stardust bitwarden_rs]$ 
+
+Build the server executable:
+
+.. note :: If that doesn't work the first time and the build failed, **just try again until it's done**.
+
+.. code-block:: console
+
+ [isabell@stardust bitwarden_rs]$ cargo build -j 1 --release --features sqlite
+
+In the next step we will download the latest build for the web vault. Check `this page`_ for the newest build number and **replace it** in the following snippet:
+
+.. note :: If you don't want to use the web-vault feature (web-app access to your vaults) for any security reasons you can skip this step. Add the following line to your ``env`` file later if you do so:
+  ``WEB_VAULT_ENABLED=false``
+
+.. code-block:: console
+ :emphasize-lines: 1,2
+
+ [isabell@stardust bitwarden_rs]$ wget https://github.com/dani-garcia/bw_web_builds/releases/download/v2.13.2/bw_web_v2.13.2.tar.gz
+ [isabell@stardust bitwarden_rs]$ tar -xvzf bw_web_v2.13.2.tar.gz
+ [isabell@stardust bitwarden_rs]$ rm -r bw_web_v*.tar.gz
+
+ 
+Generate an ``openssl -base64`` key now and save it temporarily, you'll need it in the next step.
+
+.. code-block:: console
+
+ [isabell@stardust bitwarden_rs]$  openssl rand -base64 48
+ Ig/8bZXhqVFK11F2tQZTfODO/6QpCHE3DGyCH/2Eh40xUWMFC13J6nJVPLlyU3nO
+
+Use your favourite editor to create ``~/bitwarden_rs/.env`` with the following content:
+
+.. code-block:: ini
+
+ ADMIN_TOKEN=PASTE YOUR TOKEN GENERATED ABOVE HERE # generate one with ~$ openssl rand -base64 48
+ ROCKET_PORT=62714 # your port here
+
+ SMTP_HOST=stardust.uberspace.de
+ SMTP_FROM=isabell@uber.space
+ SMTP_PORT=587
+ SMTP_SSL=true
+ SMTP_USERNAME=isabell@uber.space
+ SMTP_PASSWORD=MySuperSecretPassword
+ 
+ DOMAIN=https://isabell.uber.space
+
+Replace the mail placeholder variables with your valid IMAP credentials, otherwise the bitwarden_rs server will not be able to send you mail notifications or tokens to verify newly created user accounts.
+``SMTP_USERNAME`` and ``SMTP_PASSWORD`` must be the login data from a valid mail account. Replace the server domain with your final URL.
+
+.. note :: You can configure any type of service here, you're not limited to an uberspace IMAP user. If you prefer e.g. gmail refer to their documentations for ``SMTP_Port`` etc. accordingly.
+
+You can edit other options, look into .env.template to see a list of available options.
+
+
+Configuration
+=============
+
+Setup web backend
+-----------------
+
+.. note:: Enter the port you configured in the ``env`` file before.
```

Comment:
Users can use ports independently of each other - everyone can run their app on 9000 without collisions. Can you please adapt this and the config the use the default port (if any)?

Reply:
I am aware of the fact, that every user can use each port individually, thanks to his own nginx & the awesome web backend. But since i refer to the web-backend snippet after this section i can't insert the port directly in the code, so i decided to give the user the hint though a note.

I changed the sentence and repeated the default port from the `.env`-file above, see the latest commit.